ReadMe.md

```diff
@@ -104,17 +104,20 @@ Contributions are what make the open source community such an amazing place to b
 
 ## License
 
-Distributed under the GPL-3.0 License. See [LICENSE](https://github.com/raghavyuva/LucaMail/blob/main/LICENSE.md) for more information.
+Distributed under the GPL-3.0 License. See [LICENSE](https://github.com/raghavyuva/LucaMail/blob/main/LICENSE.md) for more information.  
+[![License](https://www.gnu.org/graphics/gplv3-with-text-136x68.png)](https://github.com/manojuppala/camKapture/blob/main/LICENSE)
 
 ## Authors
 
-* **Raghavyuva** - *Comp Sci Student* - [Raghavyuva](https://raghavyuva.com/) - *All Work*
+- **Raghavyuva** - _Comp Sci Student_ - [Raghavyuva](https://raghavyuva.com/) - _All Work_
 
 [![portfolio](https://img.shields.io/badge/my_portfolio-000?style=for-the-badge&logo=ko-fi&logoColor=white)](https://raghavyuva.com/)
 [![linkedin](https://img.shields.io/badge/linkedin-0A66C2?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/raghavyuva)
 [![twitter](https://img.shields.io/badge/twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/yuva_raghav)
 
 ## Donate
+
 If you find this project as useful then do donate so that the project is maintained up-to date.
-use  theis platforms for donation.
+use theis platforms for donation.

```

Comment:
Use this , Make Changes here

Reply:
fixed

Reply:
Remove unnecessary commits and make a single commit,

Reply:
done